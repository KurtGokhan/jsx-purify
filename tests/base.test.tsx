/** @jsxImportSource ../src/react */

describe('jsx-purify', () => {
  it('should render JSX without errors', () => {
    const jsx = <div>test</div>;
    expect(jsx).toMatchSnapshot();
  });
});
