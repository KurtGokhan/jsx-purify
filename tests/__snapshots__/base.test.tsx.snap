// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`jsx-purify > should render JSX without errors 1`] = `
{
  "$$typeof": Symbol(react.transitional.element),
  "_owner": null,
  "_store": {},
  "key": null,
  "props": {
    "children": "test",
  },
  "type": "div",
}
`;
